docker/taboot/Dockerfile

```
@@ -1,30 +1,65 @@
+FROM python:3.11-slim AS runtime
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1 \
+    UVICORN_HOST=0.0.0.0 \
     UVICORN_PORT=8000
 
-CMD ["uv", "run", "uvicorn", "llamacrawl.api.app:app", "--host", "0.0.0.0", "--port", "8000", "--reload"]
+WORKDIR /app
+
+# Install only runtime system dependencies
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)